content/using_ec2_spot_instances_with_eks/scaling/test_ca.md

```diff
@@ -170,3 +170,6 @@ completed the workshop, and before getting into the **cleanup** section.
 
  * During the workshop, we did use nodegroups that expand across multiple AZ's; There are scenarios where might create issues. Could you think which scenarios ? ([hint](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler/cloudprovider/aws#common-notes-and-gotchas)). Could you think of ways of mitigating the risk in those scenarios ? ([hint 1](https://github.com/aws-samples/amazon-k8s-node-drainer), [hint 2](https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/FAQ.md#im-running-cluster-with-nodes-in-multiple-zones-for-ha-purposes-is-that-supported-by-cluster-autoscaler))
 
+ * **Managed Spot node groups only:** AWS auto-scaling groups automatically sets the nodegroups to use the [capacity-optimized allocation strategy](https://docs.aws.amazon.com/en_pv/autoscaling/ec2/userguide/asg-purchase-options.html#asg-allocation-strategies).
```

Comment:
user proper service name: EC2 Auto Scaling groups

Reply:
Line # 173 Change "**AWS auto-scaling groups** automatically sets the nodegroups" to "EC2 Auto Scaling Group automatically sets the nodegroups"